app/views/home/index.html.erb

```diff
@@ -1 +1 @@
-<%= react_component("HomeComponent")%>
+<%= react_component("HomeComponent", props: { name: @heros_names})%>
```

Comment:
Ideally, we won't pass these as props. Our goal should be to create the proper API. And the responsibility of a component would be to communicate with that API, fetching all required data after it gets initialized. This would include the character classes :)

Creating an API (the original implementation may be very simple. Just a hash converted to json and put into response) should be one of the near future tasks. Of course, after (and if!) you have some spare time you would like to spend on playing with new things on Ragnarok :)